src/cmd/manage_nsfs.js

```diff
@@ -958,9 +968,15 @@ function validate_options_type_by_value(input_options_with_data) {
 /**
  * validate_boolean_string_value is used when the option type is boolean
  * and we wish to allow the command also to to accept 'true' and 'false' values.
+ * if has_unset_option is true allow '' value as an unset option
  * @param {boolean|string} value
+ * @param {boolean} has_unset_option
  */
-function validate_boolean_string_value(value) {
+function validate_boolean_string_value(value, has_unset_option) {
+    // unset value ''
+    if (has_unset_option && !value) {
+        return true;
+    }
```

Comment:
I would write the condition '' explicitly (`value === ''`).

Reply:
changed to have a different condition for unset items. in case it will not pass the condition it will be validated here as a regular boolean value